### PR TITLE
Update samples with more completed solution and DbUp to latest version

### PR DESCRIPTION
**Pull request by asimeonov**
Address issues #12

### CODE REVIEW

File: src/SampleApplication/Program.cs [hunk 1]
```diff
@@ -11,54 +11,100 @@ namespace SampleApplication
...
+            UpdateDatabaseFromSampleAssembly(SampleWithSuffixV4AssemblyPath, connectionString);
+            // Now update to version with scripts up to 005. Theoretically this is same assembly with just another script added
+            UpdateDatabaseFromSampleAssembly(SampleWithSuffixV5AssemblyPath, connectionString);
+            // Opps something goes wrong with the application when we deploy 005, revert to assembly with version 004
+            UpdateDatabaseFromSampleAssembly(SampleWithSuffixV4AssemblyPath, connectionString);
+        }
+
+        private static void UpdateDatabaseFromSampleAssembly(string sampleAssemblyPath, string connectionString)
+        {
+            Assembly assembly = Assembly.LoadFile(Assembly.GetExecutingAssembly().Location.Replace(SampleApplicationAssemblyPath, sampleAssemblyPath));
```
Comment (wtjones) [inline src/SampleApplication/Program.cs:103]: I had to add the various SampleScripts projects as references in this project to generate the assembly.

File: src/SampleApplication/Program.cs [hunk 2]
```diff
@@ -11,54 +11,100 @@ namespace SampleApplication
...
+        const string SampleWithSuffixV4AssemblyPath = "Samples\\FromSufix\\v4\\bin\\Debug\\net6.0\\SampleScripts.dll";
+        const string SampleWithSuffixV5AssemblyPath = "Samples\\FromSufix\\v5\\bin\\Debug\\net6.0\\SampleScripts.dll";
+        const string SampleWithFolderV4AssemblyPath = "Samples\\FromFolder\\v4\\bin\\Debug\\net6.0\\SampleScripts.dll";
+        const string SampleWithFolderV5AssemblyPath = "Samples\\FromFolder\\v5\\bin\\Debug\\net6.0\\SampleScripts.dll";
+
         public static void Main(string[] args)
         {
-            string instanceName = @".\SqlExpress";
-            // Uncomment the following line to run against sql local db instance.
+            string instanceName = @".";
```
Comment (wtjones) [inline src/SampleApplication/Program.cs:22]: I had to use this to get it to work with LocalDb: `@"(LocalDB)\MSSQLLocalDB"`. Probably not important for this change though.

File: src/SampleApplication/Program.cs [hunk 3]
```diff
@@ -11,54 +11,100 @@ namespace SampleApplication
...
 
             var connectionString = $"Data Source={instanceName};Initial Catalog=SampleApplication;Integrated Security=True;Pooling=False";
 
-            //DropDatabase.For.SqlDatabase(connectionString);
+            FromSufixSampleDeplyingTwoVersionsAndRevertingSecondDeployOfTheSameApplication(connectionString);
```
Comment (wtjones) [inline src/SampleApplication/Program.cs:26]: Minor typo: `Deplying`

**Review by wtjones — APPROVED** [review]
This change adds an important example use case. The only major issue is to resolve the project references.

File: src/SampleApplication/Program.cs (hunk 1 above)
Comment (asimeonov) [reply src/SampleApplication/Program.cs:103]: @wtjones Can you please pull the latest version clean the solution and try again. I have added pre-build events in the Sample app that will ensure that the depended libraries are build before executing them.

File: src/SampleApplication/Program.cs (hunk 3 above)
Comment (asimeonov) [reply src/SampleApplication/Program.cs]: Fixed thanks for noticing :)

File: src/SampleApplication/Program.cs (hunk 1 above)
Comment (asimeonov) [reply src/SampleApplication/Program.cs:105]: Hey, @wtjones referencing the same assembly won't showcase the actual changes. This is why I have used reflection to load the assemblies and migrate the DB, will merge the changes in this case.

**Pull request merged**